kratos/processes/compute_nodal_normal_divergence_process.h

```diff
@@ -0,0 +1,242 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ \.
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Riccardo Rossi
+//                   Vicente Mataix Ferrandiz
+//  Co-author:       Mohammad R. Hashemi
+//
+
+#if !defined(KRATOS_COMPUTE_DIVERGENCE_PROCESS_INCLUDED )
+#define  KRATOS_COMPUTE_DIVERGENCE_PROCESS_INCLUDED
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "processes/process.h"
+#include "includes/model_part.h"
+#include "includes/kratos_parameters.h"
+#include "utilities/variable_utils.h"
+#include "utilities/geometry_utilities.h"
+#include "utilities/parallel_utilities.h"
```

Comment:
please remove includes you don't use (although this one is a cool recent addition)

Reply:
Let's keep "parallel_utilities.h" for now (until finalizing it for merging). I am working on it to further improve the routines.